esti/migrate_test.go

```diff
@@ -190,7 +190,7 @@ func saveStateInLakeFS(t *testing.T) {
 	_ = createRepositoryByName(ctx, t, migrateStateRepoName)
 
 	// file is big - so we better use multipart writing here.
```

Comment:
Feel free to delete this test, it's no longer needed.

Reply:
What about the other migrate tests?